app/models/product.rb

```diff
@@ -1,3 +1,8 @@
 class Product < ApplicationRecord
+  CATEGORY = [Tent, Camp Kitchen, Sleeping Bags, Hammocks, Backpacks]
   belongs_to :user
+  validates :name, presence: true
+  validates :description, presence: true
+  validates :price, numericality: { greater_than: 1 }
+  validates :category, inclusion: { in: CATEGORY }
```

Comment:
Add presence: true to :price and :category?

Reply:
I think we should?